build/build_test.go

```diff
@@ -6,12 +6,23 @@ import (
 	"testing"
 )
 
+/*
+// TestBuild function
```

Comment:
When it comes to doc comments for test functions, we can skip those altogether; if the _name_ of the test function doesn't adequately explain what it tests, consider renaming it. Where we test several things about the same function, we would probably make those individual, named test cases (as per the Calculator exercise), where the name explains precisely what is being tested.

Reply:
Noted - makes sense, tests should not be ambiguous as to what it's doing.